src/Misc/SettingsCloudflare.php

```diff
@@ -78,13 +74,13 @@ public function isSettingsGlobalValid(): bool
 	 */
 	public function getSettingsGlobalData(): array
 	{
-		if (!$this->isOptionCheckboxChecked(self::SETTINGS_CLOUDFLARE_USE_KEY, self::SETTINGS_CLOUDFLARE_USE_KEY)) {
-			return $this->getSettingOutputNoActiveFeature();
+		if (!UtilsSettingsHelper::isOptionCheckboxChecked(self::SETTINGS_CLOUDFLARE_USE_KEY, self::SETTINGS_CLOUDFLARE_USE_KEY)) {
+			return UtilsSettingsOutputHelper::getNoActiveFeature();
 		}
 
 		return [
-			$this->getIntroOutput(self::SETTINGS_TYPE_KEY),
-			$this->settingMiscDisclamer(),
+			UtilsSettingsOutputHelper::getIntro(self::SETTINGS_TYPE_KEY),
+			UtilsSettingsOutputHelper::getMiscDisclamer(),
```

Comment:
```suggestion
			UtilsSettingsOutputHelper::getMiscDisclaimer(),
```

Reply:
can't change the function name like that, this needs to be changed on the origin. tnx for the comment I will change it

Reply:
done

Reply:
Ofc, tnx